### ISSUE TITLE
[BITV] 9.2.5.3/2.1 - The visible label "Set location for weather" does not appear in the accessible name of the button in the same form as it is overridden by the use of aria-label="Actions". In this case voice activation would fail as visible text is not in accessible name. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37018]
- [ ] Fix
- [ ] Backport

![51e36c48b52ba8aa27249843b98fe342](https://user-images.githubusercontent.com/1315170/222545040-c6064dc7-290a-40ab-bb19-4547fb9f5cc5.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-c6347c127a-v2-n1

</details>
